locksmith/src/controllers/v2/lockSettingController.ts

```diff
@@ -96,13 +96,7 @@ export const getLockSettingsBySlug: RequestHandler = async (
     const slug = request.params.slug.toLowerCase().trim()
     const settings = await lockSettingOperations.getLockSettingsBySlug(slug)
 
-    if (settings) {
-      return response.status(200).send(settings)
-    }
-
-    return response.status(404).send({
-      message: 'Settings not found for this slug',
-    })
+    return response.status(200).send(settings)
```

Comment:
always return `settings` item to avoid `404` error on network:
we use this endpoint to check if a slug is already present or not, if a slug is not present this just returns `undefined` without any network error

Reply:
Should return just empty object and make the fields optional on openAPI file.